pkgs/development/libraries/opencv/4.x.nix

```diff
@@ -368,16 +386,23 @@ stdenv.mkDerivation {
 
   passthru = {
     tests = {
-      inherit qimgv;
       inherit (gst_all_1) gst-plugins-bad;
-    } // lib.optionalAttrs (!enablePython) { pythonEnabled = pythonPackages.opencv4; };
+    }
+    // lib.optionalAttrs (!stdenv.isDarwin) { inherit qimgv; }
+    // lib.optionalAttrs (!enablePython) { pythonEnabled = pythonPackages.opencv4; }
+    // lib.optionalAttrs (stdenv.buildPlatform != "x86_64-darwin") {
+      opencv4-tests = callPackage ./tests.nix {
+        inherit enableGStreamer enableGtk2 enableGtk3 runAccuracyTests runPerformanceTests version;
+        inherit opencv4;
+        };
+      };
   } // lib.optionalAttrs enablePython { pythonPath = [ ]; };
 
   meta = with lib; {
     description = "Open Computer Vision Library with more than 500 algorithms";
     homepage = "https://opencv.org/";
     license = with licenses; if enableUnfree then unfree else bsd3;
-    maintainers = with maintainers; [ mdaiter basvandijk ];
```

Comment:
why remove mdaiter?

Reply:
He is not active anymore. In the nixpkgs and elsewhere over github, it seems. Last PR in nixpkgs in [2017](https://github.com/NixOS/nixpkgs/pulls?q=is%3Apr+author%3Amdaiter+is%3Aclosed), no activity in his github profile over the last year. Personally, I think we better to not delude ourselves and remove maintainers after some period (e.g. a year) of inactivity.

Reply:
added a comment about removing him to the commit msg so people don't think it was an accident